test/e2e/autoscale_test.go

```diff
@@ -136,7 +136,7 @@ func TestTargetBurstCapacity(t *testing.T) {
 		if err != nil {
 			return false, err
 		}
-		// We want exactly 2. Not 1, not panicing 3, just 2.
+		// We want exactly 2. Not 1, not panicking 3, just 2.
```

Comment:
I understand that the verb is `panic`. The English conjugation is `panicking`. afaik, `panicing` isn't commonly ([ngrams](https://books.google.com/ngrams/graph?content=panicking%2Cpanicing&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cpanicking%3B%2Cc0%3B.t1%3B%2Cpanicing%3B%2Cc0), [trends](https://trends.google.com/trends/explore?date=all&q=panicing,panicking)) used.

Reply:
this is more like (panic)ing as in go's `panic`.

Reply:
Yeah, I get that.

https://www.google.com/search?q=go+%22panicing%22
~298,000 results
+Did you mean: go "panicking"

https://www.google.com/search?q=go+%22panicking%22
~14,300,000 results

Reply:
this looks good `panicking`